pygmt/tests/test_grdclip.py

```diff
@@ -69,3 +72,19 @@ def test_grdclip_no_outgrid(grid, expected_grid):
     assert temp_grid.gmt.gtype == GridType.GEOGRAPHIC
     assert temp_grid.gmt.registration == GridRegistration.PIXEL
     xr.testing.assert_allclose(a=temp_grid, b=expected_grid)
+
+
+def test_grdclip_replace():
+    """
+    Test the replace parameter for grdclip.
+    """
+    grid = load_earth_mask(region=[0, 10, 0, 10])
+    npt.assert_array_equal(np.unique(grid), [0, 1])  # Only have 0 and 1
+    grid = grdclip(grid=grid, replace=[0, 2])  # Replace 0 with 2
+    npt.assert_array_equal(np.unique(grid), [1, 2])
+
+    # Test for the deprecated 'new' parameter
+    # TODO(PyGMT>=0.19.0): Remove this test for the 'new' parameter
```

Comment:
Clarify that only the test below is to be removed, not the entire unit test.

```suggestion
    # TODO(PyGMT>=0.19.0): Remove this test below for the 'new' parameter
```